types/index.d.ts

```diff
@@ -0,0 +1,16 @@
+export {};
+
+declare global {
+  interface HTMLElement {
```

Comment:
Without doing any test, this feels like it could conflict with the native HTMLElement type?